docs/guide/cli.md

```diff
@@ -49,6 +49,10 @@ export default {
   '*.{js,ts}': 'vitest related --run',
 }
 ```
+
+### `vitest bench`
```

Comment:
Why is it part of `vitest related` tip block?

Reply:
You are right, I inserted `vitest bench` into wrong place. Fixed